src/SWP/Component/MultiTenancy/Resolver/TenantResolver.php

```diff
@@ -49,21 +42,36 @@ public function __construct($domain, TenantRepositoryInterface $tenantRepository
      */
     public function resolve($host = null)
     {
-        if (null === $host) {
-            $host = self::DEFAULT_TENANT;
```

Comment:
that const should be removed from TenantResolverInterface because it is not used anywhere now afaik